### DISCUSSION TITLE
AMD Ryzen 7 6800H with Radeon Graphics - will this work

### DISCUSSION THREAD

**Discussion by BennieFun** [lllyasviel/Fooocus#3670] [Q&A]
Hi, 
Sorry, new to this. 
I'm trying to download/load Fooocus but getting the dreaded; 
RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx
Fooocus_win64_2-5-0\Fooocus_win64_2-5-0>pause
Press any key to continue . . .

I'm running a AMD Ryzen 7 6800H with Radeon Graphics laptop with 16Gb Ram

**Comment by Rogala** [comment]
Fooocus is originally designed to work with Nvidia, if you have a video card from another manufacturer or no video card at all, read the instructions on how to run on[ them](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#linux-amd-gpus).
